File: math/basic/Smallest_digit_in_number/python/smallestDigit.py
'''
Problem: To find the smallest digit in a number.
Author: Aayush Bisen
Date: 02-11-18 07:33
'''


number = int(input("Enter a number: "))
l1 = [int(i) for i in str(number)]
l1.sort()
print(l1[0],"is the smallest digit in the number.")

File: math/basic/Smallest_digit_in_number/R/smallestDigit.r
{
    n = as.integer(readline(prompt = "Enter a number :"))
    
    s = 9
    while (x > 0) {
      y = x %% 10
      if (s > y) {
        s = y
      }
      x = x %/% 10
    }
    print(paste("Smallest digit:", s))
}